components/inputs/input-date.js

```diff
@@ -195,26 +194,36 @@ class InputDate extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMixin(
 
 		this._textInput = this.shadowRoot.querySelector('d2l-input-text');
 
+		const hiddenContent = this.shadowRoot.querySelector('.d2l-input-date-hidden-text');
+		const updateHiddenContentWidth = () => {
```

Comment:
Minor: `tryUpdateHiddenContentWidth` might be a bit clearer, since it returns whether it did or not